metadata.json

```diff
@@ -2,8 +2,8 @@
   "name": "puppetlabs-reboot",
   "version": "0.1.10-SNAPSHOT",
   "author": "puppetlabs",
-  "summary": "Adds a type and provider for managing Windows system reboots.",
-  "license": "Apache License, Version 2.0",
+  "summary": "Adds a type and provider for managing system reboots.",
+  "license": "Apache-2.0",
```

Comment:
I changed it here because puppet-lint recommends Apache-2.0 for SPDX compatibility.